acinclude.m4

```diff
@@ -136,6 +136,10 @@ dnl				else
 			DRIVERS="$DRIVERS EyeboxOne${SO}"
 			actdrivers=["$actdrivers EyeboxOne"]
 			;;
+                futaba)
+                        DRIVERS="$DRIVERS futaba${SO}"
+                        actdrivers=["$actdrivers futaba"]
+                        ;;
```

Comment:
You should check if libusb is actually available. See how it is done for the picolcd driver for example.